src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ArrayCreationAndInitializer.cs

```diff
@@ -22,10 +22,11 @@ public void F()
 }
 ";
             string expectedOperationTree = @"
-IArrayCreationExpression (OperationKind.ArrayCreationExpression, Type: System.String[]) (Syntax: 'new string[1]')
+IArrayCreationExpression (OperationKind.ArrayCreationExpression, Type: System.String[], Language: C#) (Syntax: 'new string[1]')
```

Comment:
>Language: C# [](start = 88, length = 12)

It is probably not that useful to dump Language for all nodes in the tree. We could pass the expected language to the VerifyOperationTreeAndDiagnosticsForTest helper and have it verify that all nodes use it.

Reply:
Perhaps VerifyOperationTreeAndDiagnosticsForTest could even figure out what language to expect based on the syntax node, or, if it is already language specific, we can hardcode expected language in it.

---
In reply to: [141148813](https://github.com/dotnet/roslyn/pull/22309#discussion_r141148813) [](ancestors = 141148813)